go/consensus/tendermint/apps/registry/state/state.go

```diff
@@ -50,6 +51,16 @@ var (
 	//
 	// Value is CBOR-serialized roothash.ConsensusParameters.
 	parametersKeyFmt = keyformat.New(0x16)
+	// keyMapKeyFmt is the key format used for key-to-node-id map.
+	// This stores the consensus and P2P to Node ID mappings.
+	//
+	// Value is binary signature.PublicKey (node ID).
+	keyMapKeyFmt = keyformat.New(0x17, &signature.PublicKey{})
+	// certificateMapKeyFmt is the key format used for certificate-to-node-id map.
+	// This stores the hash-of-certificate to Node ID mappings.
+	//
+	// Value is binary signature.PublicKey (node ID).
+	certificateMapKeyFmt = keyformat.New(0x18, []byte{})
```

Comment:
Why `[]byte{}` instead of `&hash.Hash{}`?

Reply:
Oops, fixed! :)